src/galaxy/StarSystemGenerator.cpp

```diff
@@ -703,8 +703,11 @@ void StarSystemRandomGenerator::PickPlanetType(SystemBody *sbody, Random &rand)
 		sbody->m_radius = fixed::FromDouble(pow(sbody->GetMassAsFixed().ToDouble(), 0.5));
 	} else {
 		// Anything bigger than 200 EU masses is a Gas Giant or bigger but the density changes to decrease from here on up...
-		sbody->m_radius = fixed::FromDouble(22.6 * (1.0 / pow(sbody->GetMassAsFixed().ToDouble(), double(0.0886))));
+		sbody->m_radius = fixed::FromDouble(22.6 * (1.0 / pow(sbody->GetMassAsFixed().ToDouble(), 0.0886)));
```

Comment:
Not anything you should address here, but if we're doing planet-generation math in double-precision we might as well drop the fixed-point round-trip - either we have deterministic double-precision IEEE floats or the effort for determinism in the rest of the codebase is worthless because we're not doing the math with deterministic operations.

Reply:
OK, reverted. I read that as '0.0886 is a double and double() is casting the 0.0886 to a double which it already is'.